_docs/installation/linux.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Instalación Linux Ubuntu 20.04 LTS
+description: Instalar EVAB en Linux Ubuntu 20.04 LTS
+---
+# Instalación en Linux
+
+## Requisitos:
+
+- SFML Versión 2.5.1
+- Compilador C++ 17
+
+## Instalando requisitos
+
+- Abre una terminal y ejecutar el siguiente comando:
+  ```bash
+  $ sudo apt-get install libsfml-dev
+  ```
+- Si no tuvieras instalado el compilador MinGW(g++), abre un terminal y ejecuta el siguiente comando:

```

Comment:
Me parece que `g++` no es MinGW, es de GNU, o no? 🤔

Reply:
Efectivamente MingGW es (minimalist GNU for Windows) en Linux es solo gcc o g++ directo porque esta basado en.